handbook/communication/index.md

```diff
@@ -84,6 +84,17 @@ See "[1-1 meetings](../leadership/1-1.md)".
 
 See "[Announcements](announcements.md)".
 
+## Shortening URLs
```

Comment:
Just to check @sarah-reece (since you were involved in setting up these URLs, feel free to ask someone else), do we prefer one or another of these? If we want to always use srcgr.ph, for example, we could remove the other two listed here...

Reply:
@dadlerj @InesRoitman We've been using srcgr.ph - we secured the other two proactivey.